projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/src/utils.ts

```diff
@@ -0,0 +1,249 @@
+import { translate } from './i18n';
+import { Facebook, Mail, WordPress } from './images';
+import type { UserInfo, VerbumComments } from './types';
+
+export function classNames( ...args: Array< string | Record< string, boolean | string > > ) {
+	const result = [];
+	for ( let i = 0; i < args.length; i++ ) {
+		if ( typeof args[ i ] === 'object' ) {
+			result[ i ] = Object.keys( args[ i ] )
+				.filter( key => args[ i ][ key ] )
+				.join( ' ' );
+		} else if ( args[ i ] ) {
+			result[ i ] = args[ i ];
+		}
+	}
+	return result.join( ' ' );
+}
+
+export const serviceData = {
+	wordpress: {
+		cookieName: 'wpc_wpc',
+		name: 'WordPress.com',
+		popup: ',height=980,width=500',
+		icon: WordPress,
+		class: 'wordpress-login',
+	},
+	facebook: {
+		cookieName: 'wpc_fbc',
+		name: 'Facebook',
+		popup: ',height=650,width=750',
+		icon: Facebook,
+		class: 'facebook-login',
+	},
+	mail: {
+		name: translate( 'Email' ),
+		icon: Mail,
+		class: 'mail-login',
+	},
+};
+
+export const canWeAccessCookies = () => {
+	// Is a WordPress cookie already set and can we read it?
+	if ( document.cookie.includes( 'wpc_' ) ) {
+		return true;
+	}
+
+	// Can we set a cookie and read our own cookie?
+	document.cookie = 'verbum_test=1; SameSite=None; Secure';
+	if ( document.cookie.includes( 'verbum_test' ) ) {
+		return true;
+	}
+
+	return false;
+};
+
+/**
+ * Uses the current bundle's size and the time it took to download and excute to estimate connection speed.
+ */
+export function isFastConnection() {
+	// TODO: Need to get bundle size in webpack.
+	// Temporarily setting to true
+	return true;
```

Comment:
Temporarily setting to true until `verbumBundleSize` is fixed and added to webpack.config.js

Reply:
Removed changes to retrieve bundle size and hardcoded to 30000